SQLQuery_4.sql

```diff
@@ -0,0 +1,8 @@
+UPDATE tblEpisodes
+SET Title=CONCAT(Title,'_CANCELLED')
+WHERE EpisodeId IN (
+    SELECT e.EpisodeId
+    FROM tblEpisodes e
+    LEFT JOIN tblDoctors d ON d.DoctorId=e.DoctorId
+    WHERE d.DoctorId IS NULL
+);
```

Comment:
can we accomplish this with a join instead of a sub-query, if so, which is better?